src/v/datalake/translation/translation_probe.cc

```diff
@@ -97,6 +97,77 @@ void translation_probe::register_created_files_metrics() {
       });
 }
 
+void translation_probe::register_throughput_metrics() {
+    namespace sm = ss::metrics;
+    std::vector<sm::label_instance> labels{
+      namespace_label(_ntp.ns()),
+      topic_label(_ntp.tp.topic()),
+      partition_label(_ntp.tp.partition()),
+    };
+    _public_metrics->add_group(
+      group_name,
+      {{
+         sm::make_counter(
+           "raw_bytes_processed",
+           _raw_bytes_processed,
+           sm::description(
+             "Number of raw bytes consumed for translation that may or may not "
+             "succeed in being translated. For example, if we fail to "
+             "communicate with the coordinator preventing translation of a "
+             "batch, this metric still ticks up."),
+           labels)
+           .aggregate({
+             sm::shard_label,
+             partition_label,
+           }),
+       },
+       {
+         sm::make_counter(
+           "decompressed_bytes_processed",
+           _decompressed_bytes_processed,
+           sm::description(
+             "Bytes post-decompression consumed for translation that may or "
+             "may not succeed in being translated. For example, if we fail to "
+             "communicate with the coordinator preventing translation of a "
+             "batch, this metric still ticks up."),
+           labels)
+           .aggregate({
+             sm::shard_label,
+             partition_label,
+           }),
+       },
+       {
+         sm::make_counter(
+           "raw_bytes_translated",
```

Comment:
I don't love terms like translation leaking into the naming because I don't think it's intuitive for customers (at least at first glance), but I guess we have it in other places so that ship has sailed

Reply:
Hmm... maybe could've been "raw_bytes_consumed_for_parquet" or something? Anyway, I removed the "translated" (success) metric and reworded the processing metric description